.builds/alpine.yml

```diff
@@ -0,0 +1,17 @@
+image: alpine/latest
+packages:
+  - alsa-lib-dev
+  - libx11-dev
+  - libxrandr-dev
+  - libxcursor-dev
+  - libxinerama-dev
+  - libxi-dev
+  - mesa-dev
+  - pkgconf
+  - go
+sources:
+  - https://github.com/hajimehoshi/ebiten
+tasks:
+  - build: |
+      cd ebiten
+      go build
```

Comment:
We also need `go build ./audio` to test the audio. `go build ./...` would be better?

Reply:
I'm not sure what `./...` does. `go build . ./audio/` seems to do the job.

Reply:
... means all the defendant directories.